### PR TITLE
Add psqlodbc96 Build

### PR DESCRIPTION
**Pull request by ayohrling**
This adds a build of psqlodbc for 9.6 and removes support for EL6 builds.

@immuta/developers @immuta/devops

### CODE REVIEW

File: mock/el7-x86_64-psqlodbc.cfg [hunk 1]
```diff
@@ -1,10 +1,10 @@
 config_opts['root'] = 'el7-x86_64-psqlodbc'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
-config_opts['chroot_setup_cmd'] = 'install @buildsys-build autoconf automake libtool postgresql95-devel unixODBC-devel'
+config_opts['chroot_setup_cmd'] = 'install @buildsys-build autoconf automake libtool postgresql95-devel unixODBC-devel postgresql96-devel'
```
Comment (KyleLilly) [inline mock/el7-x86_64-psqlodbc.cfg]: Do we still need the 95-devel here?

File: mock/el7-x86_64-psqlodbc.cfg [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 config_opts['dist'] = 'el7'  # only useful for --resultdir variable subst
 config_opts['releasever'] = '7'
-config_opts['environment']['PATH'] = '/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/pgsql-9.5/bin'
+config_opts['environment']['PATH'] = '/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/pgsql-9.5/bin:/usr/pgsql-9.6/bin'
```
Comment (KyleLilly) [inline mock/el7-x86_64-psqlodbc.cfg]: Same comment related to 9.5/bin.

File: mock/el7-x86_64-psqlodbc.cfg (hunk 1 above)
Comment (ayohrling) [inline mock/el7-x86_64-psqlodbc.cfg]: This same file will support builds for the older 95 RPM and hte 96 rpm

File: mock/el7-x86_64-psqlodbc.cfg (hunk 2 above)
Comment (ayohrling) [inline mock/el7-x86_64-psqlodbc.cfg]: same

**Pull request merged**